packages/linter/src/cli.ts

```diff
@@ -53,6 +53,7 @@ export function cli(argv: string[], logger = console, cmd = "amplint") {
       /^(googlebot_desktop|googlebot_mobile|chrome_desktop|chrome_mobile)$/i,
       "googlebot_mobile"
     )
+    .option(`-n, --no-supress`, "supress passing tests in output")
```

Comment:
The description seems to contradict the parameter name (supress vs no supress).

Reply:
whoops typo. I did change this but forgot to commit the edit. Will fix